beta/azure-container-apps/ReadMe.md

```diff
@@ -45,28 +56,41 @@ Deploying 1Password SCIM Bridge on Azure Container Apps comes with a few benefit
 
 ## Deploy 1Password SCIM Bridge to Azure Container App
 
-### Automatic Azure Deployment Steps using Azure CLI
+### Azure Container App Deployment steps using Azure CLI
 
 Using the Azure Cloud Shell can be easier (the assumption for the commands below is that you are using the Cloud Shell), but this can also been done directly on your system with the Azure CLI tool with the ContainerApp Extension
 
-Both methods need the Container App Extension added to the AZ tool of choice, using `az extension add --name containerapp --upgrade`
+Both methods need the Container App Extension added to the AZ tool of choice, by running `az extension add --name containerapp --upgrade`
+
+1. Start the Azure Cloud Shell from the navigation bar of your [Azure Portal](https://portal.azure.com) or directly open the [Azure Shell](https://shell.azure.com).
+2. Add the Azure Container App extension by running the following command: `az extension add --name containerapp --upgrade`
+3. Get a list of the available locations in your Azure account: `az account list-locations -o table`, verify that the region you want to deploy on [supports Azure Container Apps](https://azure.microsoft.com/en-ca/explore/global-infrastructure/products-by-region/?regions=all&products=container-apps), take note of the name field for the desired region. 
+4. Define variables for the deployment using the following example in the Cloud Shell, _(using the bash or PowerShell syntax for the commands)_.
+
+    > **Note**
+    >
+    >The ContainerAppName can not have upper case letters, and must be between 2 and 32 characters long and start and end wuth alphanumeric, it can contain letters, numbers and hyphens.
```

Comment:
Rephrase slightly to make more clear: 
> ... it can contain lowercase letters, numerals, and hyphens. It must be between 2 and 32 characters long, and cannot start or end with a hyphen

Reply:
```suggestion
    >The ContainerAppName can contain lowercase letters, numerals, and hyphens. It must be between 2 and 32 characters long, and cannot start or end with a hyphen.
```
Applying @plttn's suggestion